Ryujinx.Graphics.Vulkan/BufferManager.cs

```diff
@@ -75,6 +115,32 @@ public BufferHandle CreateWithHandle(VulkanRenderer gd, int size, out BufferHold
             return Unsafe.As<ulong, BufferHandle>(ref handle64);
         }
 
+        public unsafe MemoryRequirements GetHostImportedUsageRequirements(VulkanRenderer gd)
+        {
+            var usage = HostImportedBufferUsageFlags;
+
+            if (gd.Capabilities.SupportsIndirectParameters)
+            {
+                usage |= BufferUsageFlags.IndirectBufferBit;
+            }
+
+            var bufferCreateInfo = new BufferCreateInfo()
+            {
+                SType = StructureType.BufferCreateInfo,
+                Size = (ulong)4096,
```

Comment:
```suggestion
                Size = 4096UL,
```
Maybe it's better to add a constant to make clear what the value means?

Reply:
This is more just to see what the requirements are for a buffer, so there's not much meaning for the size.

Reply:
I've changed this to the system page size.